src/Version/CMakeLists.txt

```diff
@@ -53,4 +53,24 @@ set_source_files_properties("${version_inc}"
              HEADER_FILE_ONLY TRUE
   )
 
-add_custom_target(OMVersion ALL DEPENDS ${version_inc})
+if (NOT DEFINED ONNX_MLIR_VERSION)
+  set(ONNX_MLIR_VERSION "0.3.0")
```

Comment:
It would be cleaner if this was defined in a separate file that is easy to track down and modify, but this is still an improvement.

Reply:
added file VERSION_NUMBER.